pom.xml

```diff
@@ -281,13 +281,13 @@
         <version.io.undertow.jastow>2.0.7.Final</version.io.undertow.jastow>
         <version.io.undertow.js>1.0.2.Final</version.io.undertow.js>
         <version.jakarta.security.enterprise>1.0.1</version.jakarta.security.enterprise>
+        <version.jakarta.validation>2.0.2</version.jakarta.validation>
```

Comment:
You also need to update HV (`version.org.hibernate.validator`) to 6.0.17.Final as it's the version that has been approved for Jakarta EE 8. It can be done in another PR but it's tightly related.

There are very few changes in it, minor bugfixes and build related improvements: https://hibernate.atlassian.net/projects/HV/versions/31761/tab/release-report-done .

Reply:
@gsmet I added a commit for that. Thanks!